app/views/coronavirus_landing_page/components/landing_page/_postcode-lookup.html.erb

```diff
@@ -0,0 +1,42 @@
+<%
+  input_label ||= "Enter your postcode"
+  input_value ||= ""
+  input_error ||= ""
+  hint_text ||= "For example SW1A 2AA"
+  button_text ||= "Find"
+  error_message ||= nil
+  error_description ||= nil
+  autocomplete ||= "postcode"
```

Comment:
```suggestion
  autocomplete ||= "postal-code"
```

Reply:
ref https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete

Reply:
I think we can probably just remove this and just put it in the attribute

Reply:
Duplicate comment (as per below)
Updated to autocomplete ||= "postal-code" - `postal-code` by default but can be passed in a different value if needed